backend/src/users/entities/user.entity.ts

```diff
@@ -45,4 +46,8 @@ export class User {
   @OneToOne<Trainee>('Trainee', (trainee) => trainee.user)
   @JoinColumn()
   trainee: Trainee;
+
+  @OneToOne<Instructor>('Instructor', (instructor) => instructor.user)
+  @JoinColumn()
+  instructor: Instructor;
```

Comment:
That's my bad it should be (please also change this for me on line 48)
```suggestion
  instructor: Instructor | null;
```